### PR TITLE
Fix g:textobj_between_no_default_mappings doesn't work

### PR DESCRIPTION
**Pull request by mityu**
Fixes https://github.com/thinca/vim-textobj-between/issues/6.

### CODE REVIEW

File: plugin/textobj/between.vim [hunk 1]
```diff
@@ -12,12 +12,14 @@ let s:save_cpo = &cpo
 set cpo&vim
 
 " Interface  "{{{1
+let g:textobj_betweenimpl_no_default_key_mappings = 1
 call textobj#user#plugin('betweenimpl', {
 \      '-': {
 \        'select-a': 'af',  '*select-a-function*': 'textobj#between#impl_select_a',
 \        'select-i': 'if',  '*select-i-function*': 'textobj#between#impl_select_i',
```
Comment (thinca) [inline plugin/textobj/between.vim:19]: We can use empty default `{lhs}`.
```suggestion
\        'select-a': [],  '*select-a-function*': 'textobj#between#impl_select_a',
\        'select-i': [],  '*select-i-function*': 'textobj#between#impl_select_i',
```

Comment (mityu) [reply plugin/textobj/between.vim:19]: I cannot find such documentation in the help of textobj-user, but it seems to work and I think this implementation is simple and better than mine. I'll apply this change.

Comment (thinca) [reply plugin/textobj/between.vim]: You can find an example for defining textobj without default key-mappings in `:help textobj-user-example-filetype`.

**Review by thinca — APPROVED** [review]
Thank you!

File: plugin/textobj/between.vim (hunk 1 above)
Comment (mityu) [reply plugin/textobj/between.vim]: No way! It's completely my fault...
Thank you.

**Pull request merged**